tests/e2e/test_building.py

```diff
@@ -226,3 +227,52 @@ def test_building_project_with_modified_protostar_toml(protostar: ProtostarFixtu
     build_dir = listdir("./build")
     assert "foo.json" in build_dir
     assert "bar.json" in build_dir
+
+
+def test_build_with_contract_names(
+    protostar: ProtostarFixture, copy_fixture: CopyFixture
+):
+    copy_fixture("cairo_0_1_mixed", "./cairo_project")
```

Comment:
If the fixture is mixed, i think the test should be mixed as well

Reply:
[c2cc61e](https://github.com/software-mansion/protostar/pull/1612/commits/c2cc61ec6b147583820b15c36cc9c2cdb0ebcfb7)